src/main/java/com/gimp/GimPlugin.java

```diff
@@ -174,6 +186,8 @@ protected void startUp()
 		{
 			panel.unload();
 		}
+
+		overlayManager.add(partyPingOverlay);
```

Comment:
Assuming the method is called `.remove`, let's invoke it on `shutDown` so we don't have a lingering overlay for users that have disabled the plugin. Not sure if this is necessary or how the overlayManager API works.